.github/workflows/test.yml

```diff
@@ -99,7 +100,8 @@ jobs:
         if: steps.cache-poetry-dependencies.outputs.cache-hit != 'true'
 
       - name: Install django ${{ matrix.django }}
-        run: poetry add django==${{ matrix.django }}
+        # Add --python ^3.9 marker because django 4.0 does not support 3.7
+        run: poetry add --python ^3.9 django@^${{ matrix.django }}
```

Comment:
oh wow, didn't know they dropped support for 3.7

I was thinking if we should drop support for 3.7, but I think for now we can keep it, we still have a number of users on 3.7:
<img width="1155" alt="Screenshot 2021-12-08 at 00 04 50" src="https://user-images.githubusercontent.com/667029/145124833-76292b59-c3ca-4af5-8f7d-f64f1d80319f.png">

source: https://pypistats.org/packages/strawberry-graphql

Reply:
Yes, and 3.7 still has 18 months before EOL. https://endoflife.date/python

Here is the discussion about which python version to support for 4.0:
https://groups.google.com/g/django-developers/c/ezUpskYAcyo/m/e0V7Ho0ZAwAJ